cfgov/v1/static/js/admin/ask_cfpb_tips.js

```diff
@@ -11,6 +11,8 @@
       populateToolbar: function (toolbar) {
         if (window.location.href.indexOf('ask_cfpb') > -1) {
           var buttonset,
+          baseClass = "answer-module",
+          textInputClass = 'richtext',
           _this = this;
           buttonset = jQuery("<span class=\"" + this.widgetName + "\"></span>");
```

Comment:
This is for the admin, not the public-facing site -- it's a jQuery plugin that adds a button to the Answer rich text field toolbar so content editors can select content to get tip styling.